docs/modules/getting-started/pages/index.adoc

```diff
@@ -67,17 +67,18 @@ A minor update (consuming changes/bugfixs on a release branch such as 3.x) is, f
 
 == Roadmap
 
-Each version is tracked as a https://github.com/orgs/opendevstack/projects[GitHub project]. The current major version is 3, the next one will be 4.
+Each version is tracked as a https://github.com/orgs/opendevstack/projects[GitHub project]. The current major version is 4, the next one will be 5.
 
-=== https://github.com/orgs/opendevstack/projects/10[4] (Target date: Early 2021)
+=== https://github.com/orgs/opendevstack/projects/14[4.2] (Target Date: End-of-Feb 2022)
+- See project page for details.
 
-Planned work (subject to change):
+=== https://github.com/orgs/opendevstack/projects/10[4] (November 2021)
 
 - OpenShift 4 support (keeping 3.11 compatibility)
 - Support for Kubernetes-native Deployment resources
 - Support for Helm 3
```

Comment:
Both `Support for Kubernetes-native Deployment resources` and `Support for Helm 3` work ONLY for the component pipeline.